Student-Unity/src/main/java/csed/swe/studentunity/Logic/RegistrationService.java

```diff
@@ -15,6 +15,7 @@ public class RegistrationService {
     public String addUser(RegistrationRequest request) {
         // check if email is already in use, if not: add user to database and send verification email
         if(userService.getUser(request.getEmail()).isEmpty()) {
+            System.out.println(request);
```

Comment:
pleas do not use printing statements it is better to use a logger if you want to show something in the backend.
I think that you were checking some thing put be careful as we do not want to pull those print lines.

Reply:
I advise you to use the sonar Lint extension in IntelliJ as it warning you that there is a code smell in that line :) 
that is just to not accumulate all code smells at the end of the milestone.

Reply:
Ok well i forgot to remove that line actually.
I will try out sonarLint